nuvlaedge/agent/telemetry.py

```diff
@@ -268,42 +268,30 @@ def update_monitors(self, status_dict):
             status_dict: dictionary containing the report for Nuvla to be updated by
             each monitor
         """
-        monitor_process_time: Dict = {}
-
-        def create_new_monitor_thread(name, telemetry: Telemetry, **kwargs):
-            th = get_monitor(name)(name, telemetry, True, **kwargs)
-            th.start()
-            return th
 
         for monitor_name, it_monitor in self.monitor_list.items():
-            self.logger.debug(f' --- Monitor: {it_monitor.name} -- '
-                                f'Threaded: {it_monitor.is_thread} -- '
-                                f'Alive: {it_monitor.is_alive()}-')
-
-            if is_thread_creation_needed(
-                    monitor_name,
-                    it_monitor,
-                    log_not_alive=(logging.INFO, 'Recreating {} thread.'),
-                    log_alive=(logging.DEBUG, 'Thread {} is alive'),
-                    log_not_exist=(logging.INFO, 'Creating {} thread.')):
-
-                self.monitor_list[monitor_name] = create_new_monitor_thread(monitor_name,
-                                                                            self)
+            self.logger.debug(f'Monitor: {it_monitor.name} - '
+                              f'Threaded: {it_monitor.is_thread} - '
+                              f'Alive: {it_monitor.is_alive()}')
+
+            if it_monitor.is_thread:
```

Comment:
I believe this was the configuration prior to my changes. We need to test it when a threaded monitor fails and stops and see if it can create a new object.